stable/atlantis/templates/statefulset.yaml

```diff
@@ -74,6 +74,12 @@ spec:
           {{- end}}
           args:
             - server
+          {{- if .Values.allowForkPRs }}
```

Comment:
@jeff-knurek I thought we were using environment variables for all the flags?

Reply:
🤦‍♂️ yeah, that aligns with the convention - wasn't thinking 

I'll submit the fix, but it'll be about a week before I can get to it

Reply:
I can do it tomorrow. Sorry, didn’t know about this.

Reply:
> I can do it tomorrow. Sorry, didn’t know about this.

thanks 👍 

---
my thoughts:
* make it a major version bump, cause anyone who might have tried using it in the meantime would break when these values get removed again
* maybe add a section to the readme for using `environment` so people _(including me)_ are less likely to try to make this kind of PR in the future :) - ie, link to: https://www.runatlantis.io/docs/server-configuration.html#environment-variables

Reply:
So just to be clear, I think we should keep the `Values` but instead of applying them via cli flags, apply them via environment variables like:
```
          {{- if .Values.allowForkPRs }}
          - name: ATLANTIS_ALLOW_FORK_PRS
             value: {{ .Values.allowForkPRs }}
          {{- if .Values.defaultTFVersion }}
           - name: ATLANTIS_DEFAULT_TF_VERSION
             value: {{ .Values.defaultTFVersion }}
          {{- end }}
```

So that wouldn't require a major bump.

Also I know it's basically the same thing but at this point we may as well keep it consistent.

Reply:
Yup, that’s what I have in mind

Reply:
hmmm, how will it work if someone already adds `ATLANTIS_ALLOW_FORK_PRS` to the `environment` value?
that would create two env vars with the same key? 
users' might then be stuck with this issue: https://github.com/kubernetes/kubernetes/issues/58477

Reply:
Opened #13321

Reply:
> hmmm, how will it work if someone already adds `ATLANTIS_ALLOW_FORK_PRS` to the `environment` value?
> that would create two env vars with the same key?
> users' might then be stuck with this issue: [kubernetes/kubernetes#58477](https://github.com/kubernetes/kubernetes/issues/58477)

Yeah, that might happen. Although, may I know what's the purpose of  the `environment` env var here. Why not be explicit and set one env var for each flag?

Reply:
> what's the purpose of the `environment` env var here

the idea is that the chart isn't tightly coupled with the application. as in if a new version of the app adds/removes env value option, then the chart needs to be updated immediately as well. it also adds for the flexibility of deploying different versions of the app that use different env values 

it seems there's already now 10 values that explicitly set the env var. I know I've run into that bug of duplicate env vars, and it's quite frustrating when you don't know about it, so maybe now that more env values are added it makes sense to make all options explicit??